app/views/edit/build-config.html

```diff
@@ -20,57 +20,90 @@
               <form class="edit-form" name="form" novalidate ng-submit="save()">
                 <div class="resource-details">
                   <div class="row">
-                    <div class="col-lg-6">
+                    <div ng-class="{
+                          'col-lg-6': advancedOptions,
+                          'col-lg-12': !advancedOptions}">
```

Comment:
Spoke with @jwforres. I think we should get rid of the 2 column layout for the build config editor and just have one column like our other editors. It's a bit strange when clicking advanced options and the layout changes.